mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -1913,6 +1920,12 @@ pub async fn run_maker_swap(swap: RunMakerSwapInput, ctx: MmArc) {
                             event.clone().into(),
                         )
                     }
+
+                    #[cfg(target_arch = "wasm32")]
+                    if event.is_error() {
```

Comment:
It's used at the tests because in WASM it's almost impossible to determine the reason why `trade_test_rick_and_morty` gets stuck.

```rust
status.status(&[&"swap", &("uuid", uuid.as_str())], &event.status_str());
```
May not to log the error status if there are multiple events.

Reply:
Ok, let's keep it then 🙂